### PR TITLE
Upgrade to ruby-2.5.8

### PR DESCRIPTION
**Pull request by magneland**
Note this is Heroku EOL on Dec 25, 2020:

https://devcenter.heroku.com/articles/ruby-support#supported-runtimes

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 version: 2 # use CircleCI 2.0
 jobs: # a collection of steps
   build: # runs not using Workflows must have a `build` job as entry point
-    parallelism: 1 # run three instances of this job in parallel
+    context: appfolio_test_context
+    parallelism: 1
     docker: # run the steps with Docker
-      - image: circleci/ruby:2.4.5-jessie-node # ...with this image as the primary container; this is where all `steps` will run
+      - image: cimg/ruby:2.5.8-node # ...with this image as the primary container; this is where all `steps` will run
```
Comment (justingaylor) [inline .circleci/config.yml:7]: Does the comment still apply?

File: .circleci/config.yml [hunk 2]
```diff
@@ -33,12 +34,8 @@ jobs: # a collection of steps
             - rails-excelsior-bundle-v2-{{ checksum "Gemfile.lock" }}
             - rails-excelsior-bundle-v2-
 
-      - run: # Install pronto/rugged/libgit2 dependencies
-          name: CMake Install
-          command: sudo apt-get install cmake
```
Comment (justingaylor) [inline .circleci/config.yml:38]: I guess cmake is already installed?

**Review by justingaylor — APPROVED** [review]
One question is whether we want to move to an even newer version of Ruby like 2.6.3? Just a thought, otherwise looks good!

File: .circleci/config.yml (hunk 2 above)
Comment (magneland) [reply .circleci/config.yml:38]: It just got moved up to line 27

File: .circleci/config.yml (hunk 1 above)
Comment (magneland) [reply .circleci/config.yml:6]: I believe so.
Although these comments are not that useful.
I think they made sense when CircleCI 2.0 was released, which moved from VMs to containers.

**Comment by magneland** [comment]
> One question is whether we want to move to an even newer version of Ruby like 2.6.3? Just a thought, otherwise looks good!

We do, but I think it's safer to do it in steps.
I would like to gradually go to latest Ruby/Rails and maybe even experiment with some brand new stuff for this app, since it's fairly trivial.
I may need to build out a Postman or Cypress integration test suite first, though.

**Review by justingaylor — APPROVED** [review]

**Comment by magneland** [comment]
[FFF: upgrade Excelsior app to Ruby 2.5.x](https://trello.com/c/THiVKQwj/105-fff-upgrade-excelsior-app-to-ruby-25x)

**Pull request merged**